File: CPP/Assignment/day12/A/Address.cpp
#include<iostream>
#include<string.h>
using namespace std;


class Address{
    private :
       string area, city;
       int pincode;
    public : 
        Address(){};
        Address(string area, string city, int pincode):area(area), city(city), pincode(pincode){};
        void Accept(){
            cout<<"area : ";cin>>area;
            cout<<"city : "; cin>>city;
            cout<<"pincode : ";cin>>pincode;
        }

        void Display() const{
            cout<<area<<", "<<city<<" - "<<pincode<<endl;
        }

};

File: OOPWJAVA/Assignment/Day7/PersonWithAddress/src/com/cdac/Address.java
package com.cdac;

public class Address {
	private String street;
	private String city;
	private String pinCode;
	
	public Address() {
		this.street = " ";
		this.city = " ";
		this.pinCode = "000-000 ";
	}
	
	public Address(String street, String city, String pinCode) {
		this.street = street;
		this.city = city;
		this.pinCode = pinCode;
	}

	@Override
	public String toString() {
		return  street + ", " + city + ", "+ pinCode ;
	}
}

